data/rupeshexe.json

```
@@ -28,6 +33,49 @@
       "name": "LinkedIn: Let's connect",
       "url": "https://linkedin.com/in/rupeshexe",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "YouTube: First Video Soon",
+      "url": "https://youtube.com/@rupeshexe",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "name": "Open Source and Random Tech Blogs",
+      "url": "https://rupeshexe.hashnode.dev/",
+      "icon": "FaHashnode"
```

Another comma `,` missing there. 

```suggestion
      "icon": "FaHashnode",
```